hydradx/model/run.py

```diff
@@ -1,36 +1,23 @@
-from cadCAD import configs
```

Comment:
I know I don't usually suggest we do this, but can we keep the cadCAD integration code here but commented out?

Reply:
or somehow in the repo, but not used... maybe in a different run.py file? not sure what makes it easiest to switch back and forth